projects/packages/jetpack-mu-wpcom/src/class-jetpack-mu-wpcom.php

```diff
@@ -136,6 +138,26 @@ public static function load_map_block_settings() {
 		wp_localize_script( 'jetpack-blocks-editor', 'Jetpack_Maps', array( 'provider' => $map_provider ) );
 	}
 
+	/**
+	 * Adds a global variable containing the where the newsletter categories should be shown.
```

Comment:
I am sorry for being pesky about the comment, but I believe you missed a word: "containing the where the newsletter" 🙈

Reply:
I'm moving too fast :D Fixed it.
Thanks for your patience 🙏